tests/preprocessing/test_highly_variable_features.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+import pytest
+
+import ehrapy as ep
+from ehrapy.preprocessing.highly_variable_features import highly_variable_features
+
+if sys.platform.startswith("win"):
```

Comment:
This is no longer required, is it?

Reply:
We could also keep it for developers who are on Windows.... But I think those should properly setup an environment anyways